src/components/blocks/CampusWeatherDetail/CampusWeatherCurrentConditions/CampusWeatherCurrentValue/CampusWeatherCurrentValue.module.scss

```diff
@@ -0,0 +1,10 @@
+.parameter {
+	flex: 1 1;
+	text-align: center;
+	.label {
+		font-size: 0.8em;
```

Comment:
Id like to stick to px for size as a standard

Reply:
commit ID: 2195cec4ac9a77e3742276d10fcc9e8616e17dc0

addressed